cc_plugin_glider/util.py

```diff
@@ -25,3 +27,103 @@ def get_sea_names():
                 buf[sea_name] = code
         _SEA_NAMES = buf
     return _SEA_NAMES
+
+
+def compare_dtype(dt1, dt2):
+    '''
+    Helper function to compare two numpy dtypes to see if they are equivalent
+    aside from endianness.  Returns True if the two are equivalent, False
+    otherwise.
+    '''
+    return eq(*(dt.kind + str(dt.itemsize) for dt in (dt1, dt2)))
```

Comment:
One problem I ran into here was that for string netCDF datatypes, the Python type is `str`, which obviously doesn't have Numpy datatype attributes.  So I'll probably make a quick fix to this edge case as a separate PR.